model/types.py

```diff
@@ -38,70 +32,107 @@
 Blocknumber = int
 Day = int
 
+class SerializableEnum(Enum):
+    def __str__(self):
+        return self.value
```

Comment:
When you have an Enum in python something like `Fiat.USD` is an instance of the `Fiat` class (which extends Enum), and. These instances of the class have a `value` property which holds the actual value of the enum, in this example, the string `"usd"`.  Here's some playing around in python I did to illustrate what's happening here:
```ipython
In [1]: from model.types import *

In [2]: Fiat.USD
Out[2]: <Fiat.USD: 'usd'>

In [3]: from enum import Enum

In [4]: class RandomEnum(Enum):
   ...:     OPTION = "hello-world"
   ...: 

In [5]: RandomEnum.OPTION
Out[5]: <RandomEnum.OPTION: 'hello-world'>

In [6]: Fiat.USD
Out[6]: <Fiat.USD: 'usd'>

In [7]: RandomEnum.OPTION.value
Out[7]: 'hello-world'

In [8]: Fiat.USD.value
Out[8]: 'usd'

In [9]: f"{Fiat.USD}"
Out[9]: 'usd'

In [10]: f"{RandomEnum.OPTION}"
Out[10]: 'RandomEnum.OPTION'
```

The reason for the SerializableEnum is in order to override the default string representation of the enum instance, because that's what's used my pandas in the postprocessing phase to create the columns in the data frame when blowing up the dictionaries. 

So, for example, in the floating supply state variable we had a dict that has keys `Crypto.CELO` and `Stable.CUSD` and if these enums didn't have the `__str__` method override than the column names would look like: `floating_supply_Crypto.CELO` and `floating_supply_Crypro.CUSD` but because I force it to use the string value instead they look like `floating_supply_celo`, `floating_supply_cusd` which is compatible with what we had before when the floating supply dict looked like `{'celo': 1000, 'cusd': 2000}`

Reply:
thanks for the detailed explanation, now I get it and this is nice